### ISSUE TITLE
Stable 2.2.0.0 Update Major Issues

### ISSUE THREAD

**Issue by Connor-Colenso** [GTNewHorizons/GT-New-Horizons-Modpack#10930]
### Your GTNH Discord Username

Colen

### Information

A stable is soon approaching, in preparation for this we are putting together a list of major issues requiring resolution prior to this. Please comment any issues you feel deserve to be on this list and we will absorb them if we feel they are essential.

### Description

- [X] Ender IO travel anchor causes players to disconnect. Unknown cause. https://github.com/GTNewHorizons/EnderIO/pull/81
- [x] Some Botania material parts inside GT generate odd recipes in. Relatively easy to fix. Especially dream/livingwood rotors as this bricks certain machines. https://github.com/GTNewHorizons/GT5-Unofficial/pull/1208
- [X] Some Botania materials need recipes. I.e. macerator for livingwood -> livingwood dust. https://github.com/GTNewHorizons/GT5-Unofficial/pull/1208
- [x] Botania creative pool needs either a recipe removal or a 9x9 recipe. https://github.com/GTNewHorizons/Botanic-horizons/pull/11
- [x] FPS lag generated from animated textures. Potentially a backport is possible, see https://bugs.mojang.com/browse/MC-132488 and https://github.com/DimensionalDevelopment/VanillaFix/tree/master/src/main/java/org/dimdev/vanillafix/textures.
- [X] Remove silly errors generated by OC. https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10663
- [x] Remove silly errors at world load, unknown cause. "ERROR: Group already exists basalt". https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/10949
- [x] Personalspace causing issues with people spawning in parallel dimensions/under bedrock and dying. Unknown cause.
- [x] Some kevlar line recipes missing a circuit, causing vital recipe conflicts with base materials.
- [x] Draconic core multi needs a UHV tiered recipe, add via TT with UHV parts not LuV. https://github.com/GTNewHorizons/TecTech/pull/77
- [x] Draconic core multi needs schematics to be obtainable somehow.
- [x] Bee breeding in the industrial apiary always has fertility 1. This massively damages bee breeding given the WA blacklist changes. https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10939
- [x] Make CAL take new tier appropriate soldering alloy. https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/369
- [X] Fix missing Electrolyzer recipes https://github.com/GTNewHorizons/GT5-Unofficial/pull/1255
- [x] Autoplace integration across all addons.
	- [x] GT https://github.com/GTNewHorizons/GT5-Unofficial/pull/1117
	- [x] TecTech https://github.com/GTNewHorizons/TecTech/pull/79
	- [x] bartworks https://github.com/GTNewHorizons/bartworks/pull/182
	- [x] GT++ https://github.com/GTNewHorizons/GTplusplus/pull/280
	- [x] KekzTech https://github.com/GTNewHorizons/KekzTech/pull/41
- [x] NEI Crashes in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11039
- [x] Put official servers back at top of server list.
- [x] Major AE Bugs https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11102, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11075
- [x] Fix industrial cutter, ABS, arc furnace structure checks https://github.com/GTNewHorizons/GTplusplus/pull/294
- [x]  Large Bronze Boiler does not form https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11114
- [x] Dangote not working correctly https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11116
- [x] Cyclotron structure is broken. https://github.com/GTNewHorizons/GTplusplus/pull/299

**Comment by hoshinokiseki** [comment]
so will you repair "Could not get provider type for dimension 32" error? this error leads to crash. even 2.1.2.4 has bettercrash mod, it still can cause another question.

**Comment by Glease** [comment]
@hoshinokiseki Please link the issue you are mentioning. If there isn't one, please create a new ticket for it.

**Comment by miozune** [comment]
I guess they mean https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10377 and similar crash when teleporting

**Comment by chochem** [comment]
@Glease not (just) teleporting. I can consistently reproduce #10377 by leaving my sp world to main menu and starting a fresh world without client restart.

**Comment by Glease** [comment]
Might be related https://github.com/GTNewHorizons/Galacticraft/pull/49

**Comment by combusterf** [comment]
Botania's *wood and *stone don't fit typical gregtech materials, and were absent from the issues that asked for compat materials. These might just as well be reverted again if they cause issues.

**Comment by chochem** [comment]
1. I think these 2 errors should probably be fixed (or hidden if harmless :P) for a stable release https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10663
2. Kevlar is still untested which might be fine to push to later as it has no use yet either. **But** I have one open ticket arising from it that affects other stuff, namely a nitrobenzene recipe with slightly different ingredients and in the LCR, that should not exist imo: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10650 . Best consult dream though before 'fixing' as I still dont know what exactly he is thinking there and he added it.

**Comment by miozune** [comment]
It is possible to fix rotor issue while keeping livingwood / livingrock materials. The problem is whether we want plates and other oreprefixes of Botania materials or not. How do you think? @Dream-Master

**Comment by combusterf** [comment]
We also have the more typical ingots and gems with all the corresponding GT versions (which seem to not be an issue), the difference being that those are actually in use.

**Comment by Steelux8** [comment]
The entire Kevlar line needs to be properly reviewed before the next stable. Even if Kevlar itself isn't used for anything, that line brought some hidden buffs and changes, like the buff to oil processing in DTs, the earlier nitrobenzene recipe, and it might have others that we haven't found yet.

**Comment by xSkewer** [comment]
![image](https://user-images.githubusercontent.com/43712386/183106294-d64c0527-6c71-4fbf-8a5a-0ed4e1947582.png)
This recipe in the Kevlar line being circuit-less conflicts with some recipes and some people's setups for other lines (mainly Polyphenylene Sulfide), so it should be removed/given a circuit before stable.

**Comment by Steelux8** [comment]
Yet another problematic recipe in the kevlar line. We need to scour the entire thing and make a list, and then put in this ticket.

**Comment by GlodBlock** [comment]
what oil buff?

**Comment by Dream-Master** [comment]
> It is possible to fix rotor issue while keeping livingwood / livingrock materials. The problem is whether we want plates and other oreprefixes of Botania materials or not. How do you think? @Dream-Master

i would keep materials

**Comment by Steelux8** [comment]
> what oil buff?

When the circuits were added to the DT recipes, for the new chemicals used in the Kevlar line, some of the new oil processing recipes were faster than the ones before, not the same speed.

**Comment by Dream-Master** [comment]
> ![image](https://user-images.githubusercontent.com/43712386/183106294-d64c0527-6c71-4fbf-8a5a-0ed4e1947582.png) This recipe in the Kevlar line being circuit-less conflicts with some people's setups for other lines (mainly Polyphenylene Sulfide), so it should be removed/given a circuit before stable.

what i see in the recipe ?

**Comment by Dream-Master** [comment]
> what oil buff?

i add a second recipe you get more out of oil (LCR recipe with 9x oil but better outputs)

**Comment by xSkewer** [comment]
> what i see in the recipe ?

This is the recipe for Sulfur Dichloride, made from Sulfur Dust and Chlorine.

**Comment by Florexiz** [comment]
We also lost most assembler rotor recipes
![image](https://user-images.githubusercontent.com/29033773/183147718-aeda31ed-4106-4f3b-b533-9f9d41e018a6.png)

**Comment by Connor-Colenso** [comment]
> We also lost most assembler rotor recipes ![image](https://user-images.githubusercontent.com/29033773/183147718-aeda31ed-4106-4f3b-b533-9f9d41e018a6.png)

Any idea why this occurred? A specific PR? I was under the impression most people used the extruder for this anyway.

**Comment by Florexiz** [comment]
Most likely related to https://github.com/GTNewHorizons/GT5-Unofficial/pull/1208/
For some reason validation fails

**Comment by Flanisch** [comment]
https://bugs.mojang.com/browse/MC-132488

Don't know how applicable this is in regards to the FPS lag. It would require the render optimizations from VanillaFix to be backported to 1.7.

**Comment by chochem** [comment]
please add to the list the wyvern/aw drac/chaotic cores are currently uncraftable. Boubou seems to be working on it and alternatively there is the option of a temporary quest for the schematics (see draft pr), but this needs to be resolved in some form before the stable version.

**Comment by miozune** [comment]
Weird, I do have the recipes. Are you on GT5 runClient? GT5 alone does not register rotors much
![2022-08-06_12 16 40](https://user-images.githubusercontent.com/40035906/183231434-9015c9ef-499a-4809-aba9-e0ca6179f1b8.png)

**Comment by chochem** [comment]
sadly I got another one: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10939
The gt industrial apiary cannot be used for breeding right now and with the new WA restrictions for other options, that is just a massive unintended nerf to bees.

**Comment by Florexiz** [comment]
> Weird, I do have the recipes. Are you on GT5 runClient? GT5 alone does not register rotors much ![2022-08-06_12 16 40](https://user-images.githubusercontent.com/40035906/183231434-9015c9ef-499a-4809-aba9-e0ca6179f1b8.png)
You cant do this on zeta with this rotor hotfix, but in sp without patch recipe still present
![image](https://user-images.githubusercontent.com/29033773/183239307-1092e855-eba8-4d4f-9ebf-0010891fbb91.png)
Still, infinity rotor for example missing assembler recipe even in sp

**Comment by Florexiz** [comment]
> Weird, I do have the recipes. Are you on GT5 runClient? GT5 alone does not register rotors much ![2022-08-06_12 16 40](https://user-images.githubusercontent.com/40035906/183231434-9015c9ef-499a-4809-aba9-e0ca6179f1b8.png)

Tbh I don't understand at this point. Now I see HSS-S recipe after client restart, but infinity still missing in pre 25

**Comment by Runakai1** [comment]
Feel like this needs attention too. Not sure why that happens. It's the same with some tooltips of controllers not showing the name of the casing 
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10966

**Comment by Steelux8** [comment]
Was the GT++/Bartworks rocket fuel auto-conversion ever fixed? I remember that holding cells in hand to complete quests for GT++ rocket fuels converted them to the other mod's version.

**Comment by chochem** [comment]
@Steelux8 nope it wasnt. there is a few such conversions either by holding the cell or by turning it back to fluid. Only the quests got changed to not demand anything anymore. see https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10847 and https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10208.

Part of it I think is something about how fluids are registered in forge (I dont understand that though), but there might be some manual fuckery between gt++ and lanth(botdustries)/bartworks going on additionally, I dont know.

**Comment by chochem** [comment]
If someone can look at this one https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10973 before stable that would be great too. pretty annoying to breed bees or build certain multis if you dont get the name of the block you need.

**Comment by chochem** [comment]
@Connor-Colenso wouldnt it be easier to just demand indalloy for all high tier circuit recipes including single block? (crystal/wetware/bio+asoc)
So just change the normal circuit assembler recipe and bartworks will probably just transfer that to CAL

**Comment by eigenraven** [comment]
The personal space issue turned out to be a problem with the multiverse plugin config which is now removed from zeta.

**Comment by chochem** [comment]
obvious addition: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11017

**Comment by Runakai1** [comment]
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11050

Think that's important too if it seems to not be an user Error.

**Comment by chochem** [comment]
I think this one would be pretty nice for stable too: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10271
I think the loading screens are manual work though right?

**Comment by Dream-Master** [comment]
> I think this one would be pretty nice for stable too: #10271 I think the loading screens are manual work though right? Some new ones would be great too. :)

ill take care when we assembled final version

**Comment by Dream-Master** [comment]
> ![image](https://user-images.githubusercontent.com/43712386/183106294-d64c0527-6c71-4fbf-8a5a-0ed4e1947582.png) This recipe in the Kevlar line being circuit-less conflicts with some people's setups for other lines (mainly Polyphenylene Sulfide), so it should be removed/given a circuit before stable.

![image](https://user-images.githubusercontent.com/2970688/185738551-e4a1e87c-c57d-4104-97db-4546458d3be1.png)
it has a circuit

**Comment by chochem** [comment]
yea Glod added that circuit

**Comment by Dream-Master** [comment]
ok all issues fixed so far beside the cell color bug

**Comment by Dream-Master** [comment]
And i add new Loading Screen Graphics

**Comment by chochem** [comment]
this still needs more testing but AE being broken is very bad for a stable: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11102, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11075

**Comment by chochem** [comment]
> Let me add that for the next stable the old EC2 quests should be removed and new quests about AE2FC should be introduced

please make a ticket propose some then

**Comment by chochem** [comment]
Many ++ structure checks were fixed by https://github.com/GTNewHorizons/GTplusplus/pull/294 so I put the checkmark there. But its absolutely possible glee8e and me missed some.

**Comment by YannickMG** [comment]
Added https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11114

**Comment by Sphyix** [comment]
This wasn't fixed yet, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10048
Any chance it can get fixed before release?
I'm able to reproduce on 29-dev, adding it now on the ticket

**Comment by Steelux8** [comment]
[mega ABS needs to be finished](https://github.com/GTNewHorizons/GTplusplus/pull/293). I've done all the structure work, and some code work, but I haven't implemented multi-amp hatches, nor have I done the balancing decisions for the multi.

**Comment by chochem** [comment]
sorry for the misclick. just wanted to write a comment:

I think megaABS can just happen after stable, no reason to delay for it that I can see.

**Comment by Steelux8** [comment]
The reason the PR was made was that Alk really wanted the ABS to have its limitations, and started the mega version to give an alternative. If the mega itself isn't finished, then that conversation about the input buses of the ABS needs to be.

**Comment by Dream-Master** [comment]
The Mega ABS can be added next version or in the dev version after 2.2.0.0

**Comment by Dream-Master** [comment]
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11236

**Comment by chochem** [comment]
since 2.2.0.0 is out I guess we can close this
